bundle-workflow/tests/test_perf_workflow/test_perf_suite.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
file name doesn't match class, either class should be TestPerfTestSuite or rename to test_performance_test_suite.py.

Reply:
Renamed the class name to `TestPerfSuite`

Reply:
I know it's weird to have "Test" twice, but the suite is called a "PerfTestSuite", so you want "TestPerfTestSuite", and `test_perf_test_suite`.